ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/freemarker/tasks/main.yml

```diff
@@ -28,3 +28,13 @@
   command: mv /tmp/freemarker-2.3.8/lib/freemarker.jar /Users/{{ Jenkins_Username }}
   when: not freemarker.stat.exists and ansible_distribution == "MacOSX"
   tags: [freemarker, adoptopenjdk]
+
+- name: Remove downloaded packages for freemarker.jar
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - /tmp/freemarker-2.3.8
```

Comment:
Maybe abstract out 2.3.8 as you have for other roles here?

Reply:
done